### PR TITLE
[CI] Replace Rackspace by https://anaconda.org/scipy-wheels-nightly

### LINKED ISSUE #2214: Rackspace does not exist anymore -> Update PRE-matrix on Travis and Azure required
All are on the title and the situation is well explained by @matthew-brett on [this page](https://github.com/matthew-brett/multibuild).

We need to update our Travis and Azure scripts and we need to figure out a solution for DIPY wheels.

### PR DESCRIPTION
**Pull request by skoudoro**
This PR should fix #2214.

As explained before, Rackspace does not exist anymore. Many packages (numpy, scipy, pandas) upload their nightly wheels to [this anaconda server. ](https://anaconda.org/scipy-wheels-nightly)

Also, DIPY uploads to this server (twice per week)  `dev` wheels from `master`.

EDIT :  ~~The only important missing package is Cython. We need to find a solution to test the future wheels of this package.~~  it is available

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@
 *.pkl
 *.orig
 *.trk
+*.pam5
```
Comment (arokem) [inline .gitignore:17]: 😄

File: .travis.yml [hunk 2]
```diff
@@ -102,12 +101,12 @@ matrix:
       dist: xenial
       env:
         - USE_PRE=1
-        - DEPENDS="$DEPENDS scipy"
+        - DEPENDS="$DEPENDS scipy statsmodels pandas scikit_learn"
```
Comment (arokem) [inline .travis.yml:104]: IIUC, this is that we test the pre-releases of all of these?

Looks good.

**Review by arokem — COMMENTED** [review]
LGTM! I had one small question about this. Otherwise, is it  ready to go from your point of view (as soon as the CI confirms)?

File: .travis.yml (hunk 2 above)
Comment (skoudoro) [reply .travis.yml:104]: Yes, and `numpy` too of course

**Comment by arokem** [comment]
Am I reading [this](https://travis-ci.com/github/dipy/dipy/jobs/389804854) correctly that we might expect compilation failures with the upcoming Cython release? Should we report this to them?

**Comment by skoudoro** [comment]
> Am I reading this correctly that we might expect compilation failures with the upcoming Cython release? Should we report this to them?

Yes, you are reading it correctly. We should check what is the ETA for this Cython release. But I think they are still working on it.

**Comment by skoudoro** [comment]
At the same time, we need to fix this future problem `cnp.npy_intp` in the declaration, and `int` during the definition. this is not clean.

**Pull request merged**